src/components/v5/common/Table/Table.tsx

```diff
@@ -333,7 +333,7 @@ const Table = <T,>({
                         {row.getVisibleCells().map((cell, index) => {
                           const renderCellWrapperCommonArgs = [
                             clsx(
-                              'flex h-full flex-col items-start justify-center p-[1.1rem] text-md',
+                              'flex h-full flex-col items-start justify-center p-4 text-md',
```

Comment:
Does this change break anything for tables elsewhere in the app? I couldn't check the activity feeds as previous actions weren't loading for me at the time of testing.